### ISSUE TITLE
Keyboard focus when returning to results panels

### ISSUE THREAD

**Issue by danielborgmann** [slynch8/10x#1507]
When returning to a Find References or Find Symbols panel, it does not get keyboard focus. This may affect other panels as well. The pattern so far seems to be that there is no focus when it has a text entry at the top.
